src/include/cache_inline.h

```diff
@@ -133,12 +133,14 @@ __wt_cache_pages_inuse(WT_CACHE *cache)
  *     Apply the cache overhead to a size in bytes.
  */
 static WT_INLINE uint64_t
-__wt_cache_bytes_plus_overhead(WT_CACHE *cache, uint64_t sz)
+__wt_cache_bytes_plus_overhead(WT_CACHE *cache, uint64_t *sz)
```

Comment:
I'd rather everything that goes in here is loaded atomically then passed in, burying the atomic here is very weird.

Reply:
I had that originally, but wasn't happy with the verbosity of all the calls:
`__wt_cache_bytes_plus_overhead(cache, __wt_atomic_load64(&cache->bytes_internal))`
vs
__wt_cache_bytes_plus_overhead(cache, &cache->bytes_internal)

This approach saves us approx. 35 additional calls to atomic load. The tradeoff is some ugliness in functions like `__wt_btree_dirty_inuse` where we pass a reference to a stack variable in to be atomically read.

Reply:
Example commit `atomic_load`ing before passing values into `__wt_cache_bytes_plus_overhead`: https://github.com/wiredtiger/wiredtiger/commit/94065fafc733ec6eb059679758c66fd59afe33b3
It's not as bad as I thought - just an additional 15 `atomic_load` calls. I have a soft preference for the current state of things but don't mind changing it if you want to push for it.

Reply:
I'll definitely push for it, the function shouldn't really go and atomic load inside of it. It should just add the overhead to the provided value.

Reply:
👍 I've added [94065fa](https://github.com/wiredtiger/wiredtiger/pull/10400/commits/94065fafc733ec6eb059679758c66fd59afe33b3) to this PR.